src/gui/virtualstudio.cpp

```diff
@@ -839,6 +839,28 @@ void VirtualStudio::completeConnection()
 
         m_connectionState = QStringLiteral("Connecting...");
         emit connectionStateChanged();
+        m_devicePtr->setServerId(m_currentStudio.id());
+        connect(m_devicePtr.get(), &VsDevice::updateNetworkStats, this,
```

Comment:
Instead of these, we should remove [this disconnect](https://github.com/jacktrip/jacktrip/blob/fa96b03dc45ba71c8f4ad1daf54c0e76780674bf/src/gui/virtualstudio.cpp#L899C1-L899C1)